code/game/objects/structures/dresser.dm

```diff
@@ -6,6 +6,18 @@
 	density = 1
 	anchored = 1
 
+/obj/structure/dresser/attackby(obj/item/P, mob/user, params)
+	if(istype(P, /obj/item/weapon/wrench))
+		to_chat(user, "<span class='notice'>You begin to [anchored ? "unwrench" : "wrench"] [src].</span>")
+		playsound(src, P.usesound, 50, 1)
+		if(do_after(user, 20, target = src))
+			to_chat(user, "<span class='notice'>You successfully [anchored ? "unwrench" : "wrench"] [src].</span>")
+			anchored = !anchored
+
+/obj/structure/dresser/deconstruct(disassembled = TRUE)
+	new /obj/item/stack/sheet/mineral/wood (get_turf(src), 10)

```

Comment:
As I explained to Cyberboss, basically all other normal objects drop their items even if destroyed rather than deconstructed (including wood based ones)